demo/webpack/prod.config.js

```diff
@@ -46,6 +46,26 @@ for (let i = 0; i < entryPoints.apps.length; i++) {
             template: 'webpack/app.template.html'
         })
     ]);
+
+    if (entryPoint.webpart) {
+        plugins = plugins.concat([
+            new HtmlWebpackPlugin({
+                inject: false,
+                name: entryPoint.name,
+                title: entryPoint.title,
+                filename: '../../../views/' + entryPoint.name + '.webpart.xml',
+                template: 'webpack/app.webpart.template.xml'
+            }),
+            new HtmlWebpackPlugin({
+                inject: false,
+                mode: 'dev',
+                name: entryPoint.name + "Dev",
+                title: entryPoint.title + " Dev",
+                filename: '../../../views/' + entryPoint.name + 'Dev.webpart.xml',
+                template: 'webpack/app.webpart.template.xml'
```

Comment:
@helenmiller16 Were you able to get the hot module reloading, dev mode, working for the webpart using this approach. I just pulled down the PR and tried it but that Dev.webpart.xml just shows as blank. I would assume that the better option might be to continue to use the view itself for HMR and then just produce the webpart.xml for the non-dev mode version.

Reply:
@cnathe thanks for taking a look! Hmm, that's odd that it's not building, worked fine for me -- that's how I've been developing the React webpart I'm working on. As you mentioned, you wouldn't be able to load it on the page if the non-dev webpart is already there.